code/modules/mob/inventory.dm

```diff
@@ -51,46 +51,24 @@
 //Finds the first available (null) index OR all available (null) indexes in held_items based on a side.
 //Lefts: 1, 3, 5, 7...
 //Rights:2, 4, 6, 8...
-/mob/proc/get_empty_held_index_for_side(side = "left", all = FALSE)
-	var/start = 0
-	var/static/list/lefts = list("l" = TRUE,"L" = TRUE,"LEFT" = TRUE,"left" = TRUE)
-	var/static/list/rights = list("r" = TRUE,"R" = TRUE,"RIGHT" = TRUE,"right" = TRUE) //"to remain silent"
-	if(lefts[side])
-		start = 1
-	else if(rights[side])
-		start = 2
-	if(!start)
-		return FALSE
-	var/list/empty_indexes
-	for(var/i in start to held_items.len step 2)
+/mob/proc/get_empty_held_index_for_side(side = LEFT_HANDS, all = FALSE)
+	var/list/empty_indexes = all ? list() : null
+	for(var/i in side to held_items.len step 2)

```

Comment:
Not a fan of making the values offsets here. That kind of codegolf just makes it harder to read. I'd prefer the offset was explicit.